### PR TITLE
php-code-sniffer 2.8.1

### PR DESCRIPTION
**Pull request by Zifius**
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-php/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/php-code-sniffer.rb [hunk 1]
```diff
@@ -3,8 +3,8 @@
 class PhpCodeSniffer < Formula
   desc "Check coding standards in PHP, JavaScript and CSS"
   homepage "http://pear.php.net/package/PHP_CodeSniffer"
```
Comment (javian) [inline Formula/php-code-sniffer.rb]: Please change this to https:// or the audit will fail.

Comment (Zifius) [inline Formula/php-code-sniffer.rb]: Yes, corrected. Though it is required only when http to https redirect is forced

**Comment by Zifius** [comment]
Can you merge it of all good @javian? It is a security release

**Comment by javian** [comment]
I'm on it now - thanks!

**Comment by Zifius** [comment]
Thanks for merging!

**Pull request closed without merging**